packages/apollo/src/commands/client/codegen.ts

```diff
@@ -90,7 +90,11 @@ export default class Generate extends ClientCommand {
     // typescript
     globalTypesFile: flags.string({
       description:
-        'By default, TypeScript will put a file named "globalTypes.ts" inside the "output" directory. Set "globalTypesFile" to specify a different path.'
+        'By default, TypeScript will put a file named "globalTypes.ts" inside the "output" directory. Set "globalTypesFile" to specify a different path. Alternatively, set "fileExtension" to modify the extension of the file'
+    }),
+    fileExtension: flags.string({
+      description:
+        'By default, TypeScript will output ".ts" files. Set "fileExtension" to specify a different file extension'
```

Comment:
Can we add an example in this description? Just to make it crystal clear whether or not someone needs to add a `.` in the extension?
```suggestion
        'By default, TypeScript will output ".ts" files. Set "fileExtension" to specify a different file extension'. example: `--fileExtension=d.ts`
```